packages/block-library/src/site-tagline/block.json

```
@@ -56,7 +56,20 @@
 		},
 		"interactivity": {
 			"clientNavigation": true
+		},
+		"__experimentalBorder": {
+			"radius": true,
+			"color": true,
+			"width": true,
+			"style": true,
+			"__experimentalDefaultControls": {
+				"radius": true,
+				"color": true,
+				"width": true,
+				"style": true
+			}
```

```suggestion
			"style": true
```

Given the dimension controls are optional here, the border controls probably should be too.